docs/conf.py

```diff
@@ -13,7 +13,7 @@
 
 import tomlkit
 
-from sed import __version__
+from sed import __version__  # type: ignore[attr-defined]
```

Comment:
Why this?

Reply:
mypy complains otherwise. It probably didn't before because this was a committed change from before

Reply:
This is not regularly checked by our pipelines. I still don't understand why there is a problem, but fine.